how2meet/db/models.py

```diff
@@ -40,9 +39,9 @@ class Guest(Base):
     id = Column(String(36), primary_key=True, index=True)
     name = Column(String(100))
     email = Column(String(100), nullable=True)
-    phone = Column(Integer(), nullable=True)
-    status = Column(String(15))
-    event_id = Column(String(36), ForeignKey("events.id"), nullable=False)
+    phone = Column(String(15), nullable=True)
+    status = Column(String(16), nullable=False)
```

Comment:
What are we indicating here? Is it whether someone has responded at all, or whether or not they're attending? Seems like it could be a `Boolean` or event a `Tuple` if we need more than 2

Reply:
Considered making it an Enum field based on the enum I defined in the forms module, but the overhead of working with the type outweighed just storing the string. If we decide to go to a boolean later, we can change it.